README.md

```diff
@@ -27,6 +27,68 @@ This chart bootstraps a [Mender](https://mender.io) deployment on a [Kubernetes]
 - Kubernetes 1.12+
 - Helm 2.11+ or Helm 3.0-beta3+
 
+## External services required
+
+This Helm chart does not install the following external services and dependencies which are required to run Mender:
+
+- mongodb
+- MinIO
+- NATS
+
+### Installing mongodb
+
+You can install mongodb using the official mongodb helm chart using `helm3`:
+
+```bash
+$ helm repo add bitnami https://charts.bitnami.com/bitnami
+$ helm repo update
+$ helm install mongodb --set "auth.enabled=false" bitnami/mongodb
```

Comment:
How about using concrete version? So in the future customers don't meet with unexpected behavior because of version differences (There is mongodb 5.0 soon)

Reply:
https://github.com/mendersoftware/mender-helm/pull/40/commits/c29feef70d0a720dc24d3481bdf4438749ff4f5a and https://github.com/mendersoftware/mender-helm/pull/40/commits/b992e9f697138a079b3b3d299518670c903a504c